### ISSUE TITLE
Accessibility Review XPS Modules

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#947]
**Scope**
The XPS wants to provide high qualitative component blueprints for Sitecore consumers including accessibility compliance.

1. Features Module 
https://prod.eu.porsche.com/?sc_lang=de-DE#ab269879e6dea8e399a4c34432b4b8d066618ee62c33cac9fbd894eafe05e62b (unter Headline “Benefits bei Porsche”)

2. Text-Media https://preview.ppn.porsche.com/germany/aboutporsche/jobs/employer/equalopportunities/
![image](https://user-images.githubusercontent.com/42934679/93435372-a3e48980-f8c9-11ea-9a2b-9cab890c6dae.png)

3. Content Navigation https://prod.eu.porsche.com/?sc_lang=de (die sticky on page navigation)
![image](https://user-images.githubusercontent.com/42934679/93435383-a810a700-f8c9-11ea-902b-2665428b7a72.png)

Acceptance Criteria
---
- [x] Modules audited
- [x] findings discussed with XPS team

Review
---

**Feature Module:**
Image has no alt-text
Add HTML native lazy load attribute -> loading=“lazy” (https://addyosmani.com/blog/lazy-loading/)
Contrast of text above image (use headline in semi-bold which also adds semantic conjunction to toggle section) and darker background shading
Button has no conjunction with toggle area —> aria-expanded=„true”
Button looses focus (opt out: https://designsystem.porsche.com/v1/#/helpers/blur-on-focus)
The detail section has also an image but it is not rendered?
Close (X) should be a HTML button element
Text flows to wide (max-length should be at 45-75 characters per line)

**Text-Media Module:**
Image has no alt-text
Use headline (< h3 >?)
Change order of image/text area in HTML

**Content-Navigation Sticky:**
Focus styling not conform
Add descriptive aria-label attribute to links, e.g.: „Skip to Benefits”
Add navigation items (buttons) to scroll on smaller screens
